static/templates/alert_word_settings_item.handlebars

```diff
@@ -11,6 +11,12 @@
                             <input type="text" name="alert_word_name" id="create_alert_word_name" class="required"
                                 maxlength=100 placeholder="{{t 'Alert Word' }}" value="" />
                         </div>
+                        <div class="alert alert-danger"" id="empty_alert_word_error" role="alert">
```

Comment:
The lint error is caused by the extra quote at the end of: `class="alert alert-danger""`